src/config/app.config.ts

```diff
@@ -17,12 +17,24 @@ export interface AppConfig {
 }
 
 const schema = Joi.object({
-  domain: Joi.string().label('HD_DOMAIN'),
+  domain: Joi.string()
+    .uri({
+      scheme: /https?/,
+    })
+    .label('HD_DOMAIN'),
   rendererOrigin: Joi.string()
+    .uri({
+      scheme: /https?/,
+    })
     .default(Joi.ref('domain'))
     .optional()
     .label('HD_RENDERER_ORIGIN'),
-  port: Joi.number().default(3000).optional().label('PORT'),
+  port: Joi.number()
+    .positive()
+    .integer()
+    .default(3000)
+    .optional()
+    .label('PORT'),
```

Comment:
Do we want to check the validity of the port number too? (`x >= 0 && x <= 65535`)

Reply:
This sounds like a good idea…